database/database.go

```diff
@@ -2,48 +2,30 @@
  * Created by VoidArtanis on 10/22/2017
  */
 
-package shared
+package database
 
 import (
-	"fmt"
+	"context"
 
-	"github.com/jinzhu/gorm"
-	// "github.com/jinzhu/gorm/dialects/mssql"
-	// "github.com/jinzhu/gorm/dialects/mysql"
-	// "github.com/jinzhu/gorm/dialects/postgres"
-	// "github.com/jinzhu/gorm/dialects/sqlite"
+	"cloud.google.com/go/firestore"
 )
 
-var db *gorm.DB
-var err error
+var client *firestore.Client
 
-/*
-dbType can be 'MySql', 'Postrges', ”
-*/
-func Init() {
-	////MySQL
-	//db, err = gorm.Open("mysql", "user:password@/dbname?charset=utf8&parseTime=True&loc=Local")
-
-	//PostgreSQL
-	db, err = gorm.Open("postgres", "host=myhost user=gorm dbname=gorm sslmode=disable password=mypassword")
-
-	////SQLite3
-	//db, err = gorm.Open("sqlite3", "/tmp/gorm.db")
-	//
-	////SQL Server
-	//db, err = gorm.Open("mssql", "sqlserver://username:password@localhost:1433?database=dbname")
-
-	if err != nil {
-		fmt.Println(err)
-	}
-
-	//db.AutoMigrate(&models.Person{})
+// Initialize the database
+func InitDatabase() error {
+	var err error
+	projectID := "pmc-website-bfa1a"
```

Comment:
i suggest doing the following instead to import the firebase_secret_key: 

```
import (
"google.golang.org/api/option"
)
```

```
projectID := "pmc-website-bfa1a"
sa := option.WithCredentialsFile("./.config/firebase_secret_key.json")
client, err = firestore.NewClient(context.Background(), projectID, sa)
```
in this case, i made a .config folder and stored the firebase secret key inside

this approach will be simpler because we don't have to do too much setup on our own, and we can rely on just cloning the repository.